main.py

```diff
@@ -1,10 +1,42 @@
+import json
 import init_django_orm  # noqa: F401
-
+from django.utils.datetime_safe import datetime
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for nickname, player_data in players_data.items():
+        # Створюємо або знаходимо обʼєкт Race
+        race, _ = Race.objects.get_or_create(
+            name=player_data["race"]["name"],
+            defaults={"description": player_data["race"]["description"]}
+        )
+
+        guild = None
+        if player_data.get("guild"):
+            guild, _ = Guild.objects.get_or_create(
+                name=player_data["guild"]["name"],
+                defaults={"description": player_data["guild"]["description"]}
+            )
+
+        Player.objects.create(
+            nickname=nickname,
+            email=player_data["email"],
+            bio=player_data["bio"],
+            race=race,
+            guild=guild,
+            created_at=datetime.now(),
```

Comment:
Consider using `django.utils.timezone.now()` instead of `datetime.now()` for the `created_at` field to ensure that the datetime is timezone-aware, which is recommended in Django applications.